lib/zipline/zip_generator.rb

```diff
@@ -15,13 +15,13 @@ def to_s
     def each(&block)
       fake_io_writer = ZipTricks::BlockWrite.new(&block)
       ZipTricks::Streamer.open(fake_io_writer) do |streamer|
-        @files.each {|file, name| handle_file(streamer, file, name) }
+        @files.each { |file, name, mtime| handle_file(streamer, file, name, mtime) }
       end
     end
 
-    def handle_file(streamer, file, name)
+    def handle_file(streamer, file, name, mtime)
```

Comment:
"options" object would be better.

Could you update the tests and README to document this optional options argument as well?

Reply:
> Could you update the tests

I didn't find any tests that used `handle_file`, `write_file` or anything else. I just saw `normalize` being tested.